### PR TITLE
Sentry Laptops In vendors

### PR DESCRIPTION
**Pull request by KoishiVibe**
# Summary
- sentry laptops now in vendors. 
- ~~some of the computer alert text has been changed to seem more mechanical and 'efficient'.~~ future pr
- lieutenant's headset now has sentry integration by default, since he doesn't have them on cameras... yet.

### CODE REVIEW

**Comment by DexterDude** [comment]
I do like Platoon Commander's getting sentry radio though, makes them more useful in a base defense situation and giving the PlatCo more things to do and more ways to assist their platoon is neat.

**Comment by KoishiVibe** [comment]
> I do like Platoon Commander's getting sentry radio though, makes them more useful in a base defense situation and giving the PlatCo more things to do and more ways to assist their platoon is neat.

tbh it sa lazyfix for not having overwatch access to sentry cameras linked up or not having a common marine sentry band for overwatch, but...
c'est la vie.

**Review by DexterDude — CHANGES_REQUESTED** [review]

File: code/datums/fluff_emails.dm [hunk 1]
```diff
@@ -3,7 +3,6 @@
 	var/title //email title
 	var/entry_text //email content
 
-
```
[the remark this answers is not in the document]
Comment (KoishiVibe) [reply code/datums/fluff_emails.dm:6]: yeah idk whats up with that I fucked up my master somehow.

File: code/modules/defenses/sentry_computer.dm [hunk 2]
```diff
@@ -289,7 +289,7 @@
 	if(length(sentry.nickname))
 		displayname = sentry.nickname
 	var/areaname = get_area(sentry)
-	var/message = "[displayname]:[areaname] lost contact."
+	var/message = "[displayname]:[areaname] NO SIGNAL."
```
[the remark this answers is not in the document]
Comment (KoishiVibe) [reply code/modules/defenses/sentry_computer.dm:292]: hmmm
funni time.

File: code/modules/defenses/sentry_computer.dm [hunk 3]
```diff
@@ -132,7 +132,7 @@
 	var/displayname = sentrygun.name
 	if(length(sentrygun.nickname))
 		displayname = sentrygun.nickname
-	var/message = "[displayname]:[get_area(sentrygun)] Engaged"
+	var/message = "[displayname]:[get_area(sentrygun)] ENGAGED"
```
[the remark this answers is not in the document]
Comment (KoishiVibe) [reply code/modules/defenses/sentry_computer.dm:135]: bleh, fine. I'll make a sentry flavor text pass one day.

**Review by Doubleumc — APPROVED** [review]
Looks good to me.

**Pull request merged**